api/views.py

```diff
@@ -45,6 +47,15 @@ class ExportInstructorLocationsView(ListAPIView):
     serializer_class = ExportInstructorLocationsSerializer
 
 
+class ExportMembersView(ListAPIView):
+    """Show everyone who qualifies as an SCF member."""
+    permission_classes = (IsAuthenticatedOrReadOnly, )
+    paginator = None  # disable pagination
+
+    queryset = get_members()
```

Comment:
`get_queryset` should be used if you want to provide, for example, different querysets based on logged in user (http://www.django-rest-framework.org/api-guide/generic-views/#attributes). It's just a different way of showing to the framework which queryset should be used.

If we end up using my proposition (https://github.com/swcarpentry/amy/pull/577/files#r44923073) then we don't need to change `ExportMembersView` at all.

QuerySets are lazy objects and if we switch to my proposition then nothing will get evaluated unless the `ExportMembersView` is visited.